.github/workflows/main.yaml

```diff
@@ -0,0 +1,95 @@
+name: Deploy on LocalStack 
+
+on:
+  push:
+    paths-ignore:
+      - 'README.md'
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule:
+    # “At 00:00 on Sunday.”
+    - cron: "0 0 * * 0"
+  workflow_dispatch:
```

Comment:
not needed?

Reply:
deleted in: https://github.com/localstack-samples/sample-cdk-kinesis-firehose-redshift/pull/3/commits/c7bb4b90b9906faeceeb60e07b5bbc03ab7161d0